static/markdowns/pronoun_frequency_analysis.md

```diff
@@ -51,8 +51,8 @@ total number of subject pronouns in a given novel that are female for each novel
 #### Overall
 Below shows the average portion of the total number of subject pronouns in a given novel that are of each type 
 
-- Female: 0.325
-- Male: 0.675
+- Female: 0.33
```

Comment:
This needs to add up to 1, so let's leave the extra significant digit in place here